zeebe/util/src/main/java/io/camunda/zeebe/util/VisibleForTesting.java

```diff
@@ -22,7 +22,7 @@
  * way.
  */
 @Target({ElementType.TYPE, ElementType.METHOD, ElementType.FIELD, ElementType.CONSTRUCTOR})
-@Retention(RetentionPolicy.SOURCE)
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
💭 Just leaving a comment for future referrence and clarity: ArchUnit tests parse compiled code, so an annotation interface must be visible after the code is compiled.

Unfortunately, when `RetentionPolicy.SOURCE` is set, the compiler ignores this annotation, and it's not available in the compiled artifacts. This is why we must set it to `RetentionPolicy.RUNTIME`.

This is acceptable, as the `zeebe-util` module is only used to implement helper/utility classes and methods that are used internally and without relevance to the public APIs.